docs/intro.md

```diff
@@ -20,9 +20,15 @@ ingesting data into Elasticsearch. A common schema helps you correlate
 data from sources like logs and metrics or IT operations
 analytics and security analytics.
 
-ECS is still under development and backward compatibility is not guaranteed. Any
-feedback on the general structure, missing fields, or existing fields is appreciated.
-For contributions please read the [Contributing Guide](CONTRIBUTING.md).
+## Maturity
+
+With ECS turning 1.0, the team will will approach improvements by following SemVer.
+This means we will only do breaking changes at major versions, and that
```

Comment:
This means that breaking changes will be limited to major versions, and that

Reply:
Deja vu 😆